src/components/HeaderUSGS.vue

```diff
@@ -17,7 +17,6 @@
             src="@/assets/usgsHeaderAndFooter/images/usgsLogo_white-01.png"
             alt="Home"
           >
-          <logo />
```

Comment:
Does this need to be `<img />`? I get a console error as is. Vue seems to think it's referencing a 'logo' component